src/components/DataTable/DataTable.jsx

```diff
@@ -322,12 +322,13 @@ const DataTable = createClass({
 									{_.map(flattenedColumns, ({ props: columnProps }, columnIndex) => (
 										<Td
 											{..._.omit(columnProps, ['field', 'children', 'width', 'title', 'isSortable', 'isSorted'])}
+											className={cx({'&-empty-cell': _.isEmpty(_.get(row, columnProps.field))})}
```

Comment:
I'm wondering if this should actually be a property on `Table.Td`, something like `isEmpty`, that way any consumers of `Table` could make use of this styling.